packages/server/graphql/mutations/helpers/generateWholeMeetingSummary.ts

```diff
@@ -0,0 +1,36 @@
+import getRethink from '../../../database/rethinkDriver'
+import OpenAIServerManager from '../../../utils/OpenAIServerManager'
+import {DataLoaderWorker} from '../../graphql'
+import isValid from '../../isValid'
+
+const generateWholeMeetingSummary = async (
```

Comment:
A couple of times I got the summary cut off at the beginning
<img width="604" alt="image" src="https://user-images.githubusercontent.com/466991/207023097-a7bb76e4-afbe-4c59-9559-9b4320cabc44.png">

<img width="558" alt="image" src="https://user-images.githubusercontent.com/466991/207023430-5901dbf0-83da-4847-ab22-2808383761af.png">

I assume this is an AI glitch?

Reply:
> I assume this is an AI glitch?

I think so as we're not slicing the content. Out of interest, did this happen with realistic meeting content or a few test reflections?

Reply:
Are we getting multiple messages and omitting some?

Reply:
@nickoferrall these were test reflections, however the content was realistic, not a random

Reply:
> Are we getting multiple messages and omitting some?

We're returning the [top choice](https://github.com/ParabolInc/parabol/blob/d7ed4a359e2fdb4b0c9c4d46760c64db43cdb3f3/packages/server/utils/OpenAIServerManager.ts#L31) which is the way that most examples I've found online handle the response. Normally, I only see one `choice` in the response. 

A small UI improvement could be making sure that the first letter is capitalised? I've also seen examples where the summary begins with a comma, so `, The team is doing great`, so I could clean that up too

Reply:
Just tried adding this, but weirdly I got worse results! All of the summaries included "second grade" after making the change, e.g. `The second-grade team did a great job and everyone should be proud of their accomplishments.`. 

I'm sure that the prompt and meeting content order (or using topic & discussion summaries instead of individual reflections, comments, and tasks) can be improved, but I'll go ahead and merge this so we can see how it behaves with real meeting content and then iterate on the prompt & text provided.